CHANGELOG.md

```diff
@@ -358,6 +358,12 @@
 
   ([PR #1443](https://github.com/alphagov/govuk-frontend/pull/1443))
 
+- Set `debounceFormSubmitTimer` at instance level instead of global
```

Comment:
Should this be in the 'fix' category?

Could we name the title so it's clear what we're fixing rather than how we fixed it?

Reply:
Makes sense, now it became a fix. Updated title, good point about it.